### DISCUSSION TITLE
[Bug] Scoop info get-date error

### DISCUSSION THREAD

**Discussion by graphixillusion** [ScoopInstaller/Scoop#5883] [Help and Questions]
#### Current Behavior
```
scoop\apps\scoop\current\libexec\scoop-info.ps1:90 car:44
+         $item.'Updated at' = $gitinfo[0] | Get-Date
+                                            ~~~~~~~~
+ CategoryInfo          : InvalidArgument: (Mon, 25 Mar 2024 17:08:22 +0000:String) [Get-Date], ParameterBindingException
+ FullyQualifiedErrorId : InputObjectNotBound,Microsoft.PowerShell.Commands.GetDateCommand
```
This error makes this line empty:
Updated at  : 

#### Expected Behavior

No error and Updated at  : with the correct date.

### System details

**Windows version:** [e.g. 7, 8, 10, 11]
Windows 11 23H2 (Working fine under Windows 10)

**OS architecture:** [e.g. 32bit, 64bit, arm64]
64bit

**PowerShell version:** [output of `"$($PSVersionTable.PSVersion)"`]
```
Major  Minor  Build  Revision
-----  -----  -----  --------
5      1      22621  2506
```

**Additional software:** [(optional) e.g. ConEmu, Git]

#### Scoop Configuration

```json
{
    "last_update":  "2024-03-26T03:46:47.2059692+01:00",
    "scoop_branch":  "master",
    "scoop_repo":  "https://github.com/ScoopInstaller/Scoop"
}

```

**Comment by rashil2000** [comment]
Which app (and bucket) are you facing this issue with?

**Comment by graphixillusion** [comment]
Actually i'm not getting this error anymore since last week. Maybe it was silently fixed in the meantime?

**Reply by rashil2000** [reply]
Huh, magic!
